pyqtgraph/graphicsItems/LinearRegionItem.py

```diff
@@ -269,7 +268,6 @@ def boundingRect(self):
         return br
         
     def paint(self, p, *args):
-        profiler = debug.Profiler()
```

Comment:
Don't these profiler assignments actually do something?

Reply:
as far as I'm aware, these are debugging tools that probably shouldn't be scattered in production code, no?

Reply:
So, I'm inclined to keep them; they're a signal that this is a _very_ performance sensitive function, and when implementing new features, it's trivial to re-enable them so they spew output in the console (either via setting environment variables, or modifying the line to add `disabled=False` and `delayed=False`).

That said; they're not crucial on their own, when profiling/debugging, the developer can re-add them as needed, and if we're really wanting to do profiling, we probably shouldn't be dependent on these buy use a tool that can capture runtimes and average them out sort of thing.

Reply:
What might be a fair compromise here is to remove the assignment for cases where profilers are not used throughout the function/method they're in such as this case.  So instead of 

```python
profiler = debug.Profiler()
```

we can have

```python
debug.profiler()
```

This serves the same thing and should make the static code checker happy.